src/main/java/gregtech/api/items/toolitem/IGTTool.java

```diff
@@ -735,27 +735,29 @@ default AoESymmetrical getAoEDefinition(ItemStack stack) {
         ));
 
         // repair info
-        if (TooltipHelper.isShiftDown()) {
-            Material material = getToolMaterial(stack);
-
-            Collection<String> repairItems = new ArrayList<>();
-            if (ModHandler.isMaterialWood(material)) {
-                repairItems.add(OrePrefix.plank.getLocalNameForItem(material));
-            } else {
-                if (material.hasProperty(PropertyKey.INGOT)) {
-                    repairItems.add(OrePrefix.ingot.getLocalNameForItem(material));
-                } else if (material.hasProperty(PropertyKey.GEM)) {
-                    repairItems.add(OrePrefix.gem.getLocalNameForItem(material));
+        if (!tagCompound.getBoolean(UNBREAKABLE_KEY)) {
+            if (TooltipHelper.isShiftDown()) {
+                Material material = getToolMaterial(stack);
+                String materialName = I18n.format(getToolMaterial(stack).getUnlocalizedName());
+
+                Collection<String> repairItems = new ArrayList<>();
+                if (ModHandler.isMaterialWood(material)) {
+                    repairItems.add(I18n.format("item.material.oreprefix.plank", materialName));
+                } else {
+                    if (material.hasProperty(PropertyKey.INGOT)) {
+                        repairItems.add(I18n.format("item.material.oreprefix.ingot", materialName));
```

Comment:
Using the prefix translation key here (and in the other places) will cause issues with specific material lang overrides. For example, Rubber Ingots have an override to be called "Rubber Bar." In the tooltip, they will incorrectly show as "Rubber Ingot" in this case.